docs/user-guide/apis/v1alpha1.md

```diff
@@ -679,7 +679,7 @@ _Appears in:_
 | `hardware` _[VirtualMachineClassHardware](#virtualmachineclasshardware)_ | Hardware describes the configuration of the VirtualMachineClass attributes related to virtual hardware.  The configuration specified in this field is used to customize the virtual hardware characteristics of any VirtualMachine associated with this VirtualMachineClass. |
 | `policies` _[VirtualMachineClassPolicies](#virtualmachineclasspolicies)_ | Policies describes the configuration of the VirtualMachineClass attributes related to virtual infrastructure policy.  The configuration specified in this field is used to customize various policies related to infrastructure resource consumption. |
 | `description` _string_ | Description describes the configuration of the VirtualMachineClass which is not related to virtual hardware or infrastructure policy. This field is used to address remaining specs about this VirtualMachineClass. |
-| `configSpec` _RawExtension_ | ConfigSpec describes additional configuration information for a VirtualMachine. 
+| `configSpec` _integer array_ | ConfigSpec describes additional configuration information for a VirtualMachine. 
```

Comment:
Does this need to be updated?

Reply:
It was. That's how it got generated. If it wasn't updated then the GH action linter check wouldn't pass. The API generator clearly has an issue. Anyway, there's never any need to comment on generated assets as the GH action that validates generated assets are up-to-date would fail if they were not updated.